src/main/java/www/purple/mixxy/controllers/ComicController.java

```diff
@@ -17,48 +23,85 @@
 public class ComicController {
 
 	@Inject
-    private ComicDao comicDao;
-	
-  public Result remix() {
-    return Results.TODO();
-  }
-
-  public Result edit() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result newWork() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result likes() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result remixes() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result root() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result parent() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result delete() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result update() {
-    return Results.TODO();
-  }
+	private ComicDao comicDao;
+
+	@SuppressWarnings("PMD.DataflowAnomalyAnalysis")
+	public Result remix() {
+		return Results.TODO();
+	}
+
+	public Result edit() {
+		return Results.TODO();
+	}
+
+	public Result comicShow(@PathParam("id") Long id) {
```

Comment:
True, but I'm not so sure on how I can implement that in the DAO with url strings. If you can help me out with that I would appreciate it.